src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -446,6 +447,15 @@ function ComposerWithSuggestions({
         textInputRef.current.blur();
     }, []);
 
+    useEffect(() => {
+        if (!shouldAutoFocus && !willBlurTextInputOnTapOutside) {
```

Comment:
@jeet-dhandha Can you explain why this is here?

Reply:
To scroll to bottom for web, mWeb, desktop. If its android/iOS below code will not be triggered.

Reply:
I think it can be removed, I don't see any reason not to scroll to bottom on all devices that are supported.

Reply:
So you are saying I should remove whole `    if (!shouldAutoFocus && !willBlurTextInputOnTapOutside) {` check ?

Reply:
@jjcoffee

Reply:
Ideally it shouldn’t.

Reply:
I'm questioning why the check is there. I'd expect you to understand why you've added the code you are providing :smile:

Reply:
- Firstly `!shouldAutoFocus` check was there to not scroll to bottom if composer is not focused.
- Then after your suggestion to scroll all `mWeb` devices to bottom even if not focused. Thus added check `!shouldAutoFocus && !willBlurTextInputOnTapOutside`.
- But after you pointed out that this code will work for every platform without causing issues, thus i will remove the checks.